from . import config
from . import MyExceptions


if __name__ == '__main__':
    pass
